.gitignore

```diff
@@ -10,6 +10,9 @@ out
 # web site build
 site/site
 
+# vscode
+.project
```

Comment:
We can also consider getting these changes in separately, which would be quick.

Reply:
I guess better to do it in a separate PR